README.md

```diff
@@ -1,109 +1,171 @@
 
 Add a complete and customizable user authentication system for your API project.
 
-## Demo project
-
-https://github.com/nodes-vapor/jwt-keychain-demo
 
 ## 📦 Installation
 
 Update your `Package.swift` file.
 
 ```swift
-.package(url: "https://github.com/nodes-vapor/jwt-keychain.git", from: "1.0.0")
+.package(url: "https://github.com/nodes-vapor/keychain.git", from: "1.0.1")
 ```
 ```swift
 targets: [
     .target(
         name: "App",
         dependencies: [
             ...
+            .product(name: "Keychain", package: "keychain"),
+
             "Keychain"
         ]
     ),
     ...
 ]
 ```
+## Usage
+### Setup 
+```swift
+app.keychain.configure(
+    signer: JWTSigner(
+        algorithm: TestJWTAlgorithm(name: UserAccessKeychainConfig.jwkIdentifier.string)
+    ),
+    config: UserAccessKeychainConfig()
+)
+app.keychain.configure(
+    signer: JWTSigner(
+        algorithm: TestJWTAlgorithm(name: UserRefreshKeychainConfig.jwkIdentifier.string)
+    ),
+    config: UserRefreshKeychainConfig()
+)
+app.keychain.configure(
+    signer: JWTSigner(
+        algorithm: TestJWTAlgorithm(name: UserResetKeychainConfig.jwkIdentifier.string)
+    ),
+    config: UserResetKeychainConfig()
+)
+```
 
-## Getting started 🚀
-
-TODO
-
-### Resources
-
-Copy package resources:
-
-Move the content of `Keychain/Resources/Views` into the `Resources/Views` folder of your project. Unfortunately there's no convenient to this at the moment, but one option is to download this repo as a zip and then move the folders into the root of your project. Remember to check that you're not overwriting any files in your project.
-
-See `https://github.com/vapor/jwt` to learn more about signing.
-
-### Usage
-
+#### JWTPayload
 ```swift
+import JWT
 import Keychain
-```
+import Vapor
+
+struct UserJWTPayload: KeychainPayload {
+    let exp: ExpirationClaim
+    let sub: SubjectClaim
 
-### Token Generator Command
-In order to generate password reset tokens for users add the following to `droplet.json`'s `commands`: `"keychain:generate_token"`. Then you can create a token like so:
+    init(expirationDate: Date, user: User) {
+        self.exp = .init(value: expirationDate)
+        self.sub = .init(value: user.id)
+    }
 
-> `drop --run keychain:generate_token user@email.com`
+    func findUser(request: Request) -> EventLoopFuture<User> {
+        request.eventLoop.future(request.testUser).unwrap(or: TestError.userNotFound)
+    }
 
-## Tokens
+    func verify(using signer: JWTSigner) throws {
+        // don't verify anything since we're not testing the JWT package itself
+    }
+}
+```
+#### KeychainConfigs
 There are three types of tokens used by Keychain: refresh tokens, API access tokens, and password reset tokens.
 
 Both refresh and access tokens should be included in the `Authorization` header for each request they are needed for, as follows: `Authorization: Bearer TOKEN` (where `TOKEN` is replaced with the actual token string).
```

Comment:
This used to be true that only these token needed to be in the header. Now we're not opinionated anymore so I think we can just remove this sentence.